### PR TITLE
Box theatre starboard vent/scrubber fix

### PR DESCRIPTION
**Pull request by bunny232**
## About The Pull Request
Moves a vent and scrubber down 5 pixels

## Why It's Good For The Game
Things looking nice is nice.

## Changelog
:cl:
fix: moved a scrubber and vent down 5 pixels
/:cl:

### CODE REVIEW

**Comment by bunny232** [comment]
Fun fact east facing vents/scrubbers spawn with pixel_y set to 5 for me aaaaaaaaaa

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -53392,6 +53390,13 @@
 	},
 /turf/open/floor/carpet,
 /area/crew_quarters/cryopod)
+"dzQ" = (
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on{
+	dir = 4;
+	pixel_y = 0
```
Comment (Ghommie) [inline _maps/map_files/BoxStation/BoxStation.dmm]:
```suggestion
	dir = 4
```

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 2]
```diff
@@ -56158,7 +56131,7 @@
 "oyz" = (
 /obj/machinery/atmospherics/components/unary/vent_pump/on{
 	dir = 4;
-	pixel_y = 5
+	pixel_y = 0
```
Comment (Ghommie) [inline _maps/map_files/BoxStation/BoxStation.dmm:56134]:
```suggestion
```

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 3]
```diff
@@ -56158,7 +56131,7 @@
 "oyz" = (
 /obj/machinery/atmospherics/components/unary/vent_pump/on{
 	dir = 4;
```
Comment (Ghommie) [inline _maps/map_files/BoxStation/BoxStation.dmm:56133]:
```suggestion
	dir = 4
```

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 4]
```diff
@@ -57562,7 +57553,7 @@
 "tWj" = (
 /obj/machinery/atmospherics/components/unary/vent_scrubber/on{
 	dir = 4;
-	pixel_y = 5
+	pixel_y = 0
```
Comment (Ghommie) [inline _maps/map_files/BoxStation/BoxStation.dmm:57556]:
```suggestion
```

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 5]
```diff
@@ -57562,7 +57553,7 @@
 "tWj" = (
 /obj/machinery/atmospherics/components/unary/vent_scrubber/on{
 	dir = 4;
```
Comment (Ghommie) [inline _maps/map_files/BoxStation/BoxStation.dmm:57555]:
```suggestion
	dir = 4
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
You can add the suggestions to a batch and apply them in a single commit

**Comment by bunny232** [comment]
> You can add the suggestions to a batch and apply them in a single commit

Neat

**Review by Ghommie — APPROVED** [review]

**Pull request merged**